src/core/platformutilities.cpp

```diff
@@ -133,3 +135,7 @@ bool PlatformUtilities::checkWriteExternalStoragePermissions() const
   return true;
 }
 
+QString PlatformUtilities::insertLinks( const QString &string )
```

Comment:
Adding QStringUtils to PlatformUtilities also feels weird. Because QStringUtils is not a QObject, I either need to create a new strinutils wrapper, or put it in already existing utils.

@m-kuhn what would make sense here?

Reply:
PlatformUtilities are meant for things that need different implementations on different platforms. E.g. camera, file browser, ...

Instead, it would be good to register `QgsStringUtils` as singleton in qgismobileapp.cpp, so in qml you can directly do something like `StringUtils.insertLinks( text ).

Reply:
This will need a modification of QgsStringUtils in QGIS core --> i.e. it needs to be a Q_GADGET and the method needs to be Q_INVOKABLE